SocialCareCaseViewerApi.Tests/V1/Gateways/MashReferralGatewayTests/UpdateReferralTests.cs

```diff
@@ -41,51 +41,59 @@ public void UpdatingMashReferralThrowsMashReferralNotFoundExceptionWhenGetInfras
         }
 
         [Test]
-        public void UpdatingMashReferralThrowsMashReferralStageMismatchExceptionWhenRequestUpdateIsForScreeningDecisionAndReferralIsNotInScreeningStage()
+        public void
+            UpdatingMashReferralThrowsMashReferralStageMismatchExceptionWhenRequestUpdateIsForScreeningDecisionAndReferralIsNotInScreeningStage()
```

Comment:
Are we inserting a newline here because of the length of the test name? Is this from running the linter?

Wondering if could use a [description attribute](https://docs.nunit.org/articles/nunit/writing-tests/attributes/description.html) in this case so the the test function name is shorter and we can keep a consistent styling for the tests? 🤔 
e.g
```c#
[Test, Description("Updating Mash Referral Throws Mash Referral Stage Mismatch Exception")]
    public void WhenRequestIsForScreeningDecisionAndReferralIsNotInScreeningStage()
```

Reply:
Up for discussion with rest of the team, I personally think idea is great since it is difficult to read the test names.

Reply:
Yep. I agree with a team discussion 👍🏾